FileXT/common.h

```diff
@@ -4,26 +4,49 @@
 
 // Configure this when building locally to enable more verbose logging.
 // By default verbose logging is enabled in debug builds.
-#define ENABLE_LOG_VERBOSE !defined(NDEBUG)
+#if !defined(NDEBUG)
+#define ENABLE_LOG_VERBOSE
+#endif
 
 // Debug logging macros
 #ifndef NDEBUG
-extern FILE* gLogFile;
-#define LOG(...) fprintf(gLogFile?gLogFile:gLogFile=stderr, __VA_ARGS__); fflush(gLogFile)
+#define LOG(...) log(__VA_ARGS__)
 #else
 #define LOG(...)
 #endif
 
-#if ENABLE_LOG_VERBOSE && !defined(NDEBUG)
-#define LOG_VERBOSE(...) fprintf(gLogFile?gLogFile:gLogFile=stderr, "FileXT: "); fprintf(gLogFile, __VA_ARGS__ ); fprintf(gLogFile, "\n"); fflush(gLogFile)
-#elif ENABLE_LOG_VERBOSE
-#define LOG_VERBOSE(...) fprintf(stderr, "FileXT: "); fprintf(stderr,  __VA_ARGS__ ); fprintf(stderr, "\n"); fflush(stderr)
+#if ENABLE_LOG_VERBOSE
```

Comment:
Do we want `#if` here or maybe `#ifdef`?

Reply:
Previously ENABLE_LOG_VERBOSE was resolved to true/false, it seems. Now it's just defined.

Reply:
Gotcha, MSVC appears to be more relaxed about this.
I will updated this.